### DISCUSSION TITLE
How to stop execution of all Scheduled and manual dag run's in Airflow

### DISCUSSION THREAD

**Discussion by SRID45** [apache/airflow#37537] [General]
Hi Team,

    during the deployment window time, we are planning to hold the Airflow dag executions for both scheduled and manual dag runs.

    in our airflow server we are having many dags, is there any way to stop execution of all Scheduled and manual dag run's in Airflow.

    Airflow version we are using Version: [v2.7.1](https://pypi.python.org/pypi/apache-airflow/2.7.1)

Thanks.

**Comment by potiuk** [comment]
See this comment: https://github.com/apache/airflow/discussions/37531#discussioncomment-8522479
